p/scripts/extra.js

```diff
@@ -239,6 +199,85 @@ function init_configuration_alert() {
 		};
 }
 
+const init_draggable_list = function() {
+	let source;
+	const draggableList = document.querySelector('.draggableList');
+	const addMarker = (position, element) => {
+		const hr = draggableList.querySelector('hr.drag-drop-marker');
+		if (null === hr) {
+			element.insertAdjacentHTML(position, '<hr class="drag-drop-marker" />');
+		}
+	};
+	const removeMarker = () => {
+		const hr = draggableList.querySelector('hr.drag-drop-marker');
+		if (null !== hr) {
+			hr.remove();
+		}
+	};
+
+	draggableList.addEventListener('dragstart', event => {
```

Comment:
Not needed for `integration.js`, but in `main.js` and `extra.js`, could you please keep using an IE11-compatible syntax for a few more months? So no arrow functions here. Otherwise, as this code is not an essential feature, it could also be fine to move it to e.g. `integration.js`.

Reply:
@Alkarex I've extracted the draggable feature in its own file. Later we could merge it back to `extra.js`